src/Components/Facility/ConsultationForm.tsx

```diff
@@ -820,7 +827,7 @@ export const ConsultationForm = (props: any) => {
               </div>
 
               <div id="ip_no-div">
-                <InputLabel id="refered-label">IP number</InputLabel>
+                <InputLabel id="refered-label">IP number*</InputLabel>
```

Comment:
@khavinshankar You can use the `required` prop
https://mui.com/material-ui/api/input-label/

Reply:
@developedBySJ  I did think of that, but do `required` attribute help with user friendly message?

Reply:
@khavinshankar What do you mean by user-friendly message?